internal/file/finder_test.go

```diff
@@ -185,6 +186,92 @@ func TestGetGroupsPIP(t *testing.T) {
 
 }
 
+func TestGetGroupsAllExcluded(t *testing.T) {
+	setUp(true)
+	path := "testdata/misc"
+	const nbrOfGroups = 0
+
+	oldStdout := os.Stdout
+	read, write, _ := os.Pipe()
+	os.Stdout = write
+
+	fileGroups, err := finder.GetGroups(
+		DebrickedOptions{
+			RootPath:     path,
+			Exclusions:   []string{"**/**"},
+			Inclusions:   []string{},
+			LockFileOnly: false,
+			Strictness:   StrictAll,
+		},
+	)
+
+	write.Close()
+	os.Stdout = oldStdout
+	var buf bytes.Buffer
+	io.Copy(&buf, read)
+
+	assert.NoError(t, err)
+	assert.Equalf(t, nbrOfGroups, fileGroups.Size(), "failed to assert that %d groups were created. %d was found", nbrOfGroups, fileGroups.Size())
+
+	// Define the expected output
+	expectedStart := "The following files were excluded, resulting in no dependency files found."
+	expectedEnd := "Please change the inclusion and exclusion options if an important file or directory was missed."
+	expectedExcludedFile := "requirements.txt"
+
+	// Compare the actual output to the expected output
+	actualString := buf.String()
+	if !strings.Contains(actualString, expectedStart) {
+		t.Errorf("Expected %q but got %q", expectedStart, actualString)
+	}
+	if !strings.Contains(actualString, expectedEnd) {
+		t.Errorf("Expected %q but got %q", expectedEnd, actualString)
+	}
+	if !strings.Contains(actualString, expectedExcludedFile) {
+		t.Errorf("Expected %q but got %q", expectedExcludedFile, actualString)
+	}
+}
+
+func TestGetGroupsAllExcludedByStrictness(t *testing.T) {
+	setUp(true)
+	path := "testdata/misc"
+	const nbrOfGroups = 0
+
+	oldStdout := os.Stdout
+	read, write, _ := os.Pipe()
+	os.Stdout = write
```

Comment:
this is identical to `TestGetGroupsAllExcluded`? Break out to a separate function please :)